integrations/using-integrations/aws-s3.md

```diff
@@ -33,7 +41,5 @@ Aqueduct does not eagerly validate whether or not your file exists. In order to
 In order to save an aritfact to a database, you can call `.save()` on the artifact. `.save()` expects information about which system to write the artifact to and what configuratin parameters are necessary. This configuration can be generated by calling `.config` on your integration object. For AWS S3, the following parameters are required:
 
 * `filepath`: The path in the S3 bucket to which this file should be written.
-* `format`: `csv`, `parquet`, or `json`
-  * `csv`: The Artifact will be converted to a CSV file (using the Pandas `to_csv` function) before writing it to `filpath`&#x20;
```

Comment:
Why was this removed?

Reply:
The original version seems unnecessarily detailed.

Reply:
IMO, I don't think detail in documentation is a bad thing, unless it's misleading or inaccurate.

Reply:
Things like `to_csv ` is an implementation detail, which I think we don't have to reveal here for the user? No strong opinion though if you want it back.